doc/en/source/performanceNew.rst

```diff
@@ -24,91 +24,266 @@ Throughput and latency to create new entities
 --------------------------------------------------
 
 **Create**
+************
 
 .. figure:: figures/imgUpsertCreate.png
 
 .. figure:: figures/imgUpsertCreate2.png
 
-**update**
+.. figure:: figures/createdata.png
+
+**Analysis of Graphs**:
+The above graphs are plotted against Create API in Fogflow. The Y axis in graph represents response time and X axis represents timestamp to gnerate real time environment. The line parallel to X axis is the mean of response time for all the entities created using Create API. The graph shows the response time in contrast to each and every request that is handled by Fogflow broker. 
```

Comment:
this heading should be above the graph in all graphs then paste the graph, then tell about the result table and then paste the table.

Reply:
Incorporated Comments